ee/codegen/src/context/node-context/inline-prompt-node.ts

```diff
@@ -12,12 +12,17 @@ export class InlinePromptNodeContext extends BaseNodeContext<InlinePromptNodeTyp
   baseNodeDisplayClassName = "BaseInlinePromptNodeDisplay";
 
   protected getNodeOutputNamesById(): Record<string, string> {
+    const jsonOutput = this.nodeData.outputs?.find(
```

Comment:
[oos] long term, when we're done normalizing all of our nodes, we'll be able to remove all of these methods from the node class specific contexts and just return `this.nodeData.outputs.map((output) => [output.id, output.name])`